tutorials/ComplexArithmetic/ComplexArithmetic.ipynb

```diff
@@ -0,0 +1,623 @@
+    "\n",
+    "**Input:** A complex number $x = a + bi$, represented as a tuple `(a, b)`.\n",
+    "\n",
+    "**Goal:** Return the complex number $e^x = e^{a + bi} = g + hi$, represented as a tuple `(g, h)`.\n",
+    "\n",
+    "> [Python's trigonometric functions](https://docs.python.org/3/library/math.html#trigonometric-functions)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@exercise\n",
+    "def complex_exp(x : Complex) -> Complex:\n",
+    "    return # ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <span style=\"color:blue\">Exercise 8</span>*: Complex powers of real numbers.\n",
+    "\n",
+    "**Inputs:**\n",
+    "\n",
+    "1. A non-negative real number $r$.\n",
+    "2. A complex number $x = a + bi$, represented as a tuple `(a, b)`.\n",
+    "\n",
+    "**Goal:** Return the complex number $r^x = r^{a + bi} = g + hi$, represented as a tuple `(g, h)`.\n",
+    "\n",
+    "> Remember, you can use functions you have defined previously\n",
+    "\n",
+    "<br/>\n",
+    "<details>\n",
+    "    <summary><strong>Need a hint? Click here</strong></summary>\n",
+    "    You can use the fact that $r = e^{\\ln r}$ to convert exponent bases. Remember though, $\\ln r$ is only defined for positive numbers.\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@exercise\n",
+    "def complex_exp_real(r : float, x : Complex) -> Complex:\n",
+    "    return # ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Polar coordinates\n",
+    "\n",
+    "Using complex exponentiation, we now have a different way to represent complex numbers geometrically. Using the fact that $e^{ix}$ is always on the unit circle, we can represent complex numbers using **polar coordinates**. In a polar coordinate system, a point is represented by two numbers: its direction from origin, represented by an angle from the $x$ axis, and how far away it is in that direction.\n",
+    "\n",
+    "Another way to think about this is that we're taking a point that is $1$ unit away (which is on the unit circle) in the specified direction, and multiplying it by the desired distance. And to get the point on the unit circle, we can use $e^{ix}$.\n",
+    "\n",
+    "A complex number of the format $r \\cdot e^{i\\theta}$ will be represented by a point which is $r$ units away from the origin, in the direction of $\\theta$.\n",
+    "\n",
+    "![Polar Coordinates Visual Explanation](img/complex_polar.png)\n",
+    "\n",
+    "Sometimes $\\theta$ will be referred to as the number's **phase**."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <span style=\"color:blue\">Exercise 9</span>: Polar conversion.\n",
```

Comment:
"Polar conversion" can mean both conversion from polar and to polar, I'd be a bit more specific here:
```suggestion
    "### <span style=\"color:blue\">Exercise 9</span>: Cartesian to polar conversion.\n",
```